tests/storage/databases/main/test_lock.py

```diff
@@ -28,7 +28,7 @@ def test_simple_lock(self):
         """
         # First to acquire this lock, so it should complete
         lock = self.get_success(self.store.try_acquire_lock("name", "key"))
-        self.assertIsNotNone(lock)
+        assert lock is not None
```

Comment:
I find it quite annoying that mypy doesn't handle `assertIs(Not)None` the same way...

python/mypy#5088 for cross-refs.